mod/app/src/main/java/bttv/SplitChat.java

```diff
@@ -59,11 +61,28 @@ public static void setBackgroundColor(int position, RecyclerView.ViewHolder view
         Context context = view.getContext();
 
         // make sure we only change chat message items
+        // Other elements like redeems or Sub Anniversaries are ConstraintLayouts
+
+        // If view is a LinearLayout, check the childrens to find chat_message_item
+        if(view instanceof LinearLayout) {
+            Log.d(TAG, "view is LinearLayout: " + view.toString());
+            LinearLayout linearLayout = (LinearLayout) view;
+            for (int j = 0; j < linearLayout.getChildCount(); j++) {
+                View nestedChild = linearLayout.getChildAt(j);
+                if (nestedChild.getId() == ResUtil.getResourceId(context, "chat_message_item", "id")) {
```

Comment:
before the loop (before the `if` even), initialize a variable `chatMessageItemResId` with `ResUtil.getResourceId(context, "chat_message_item", "id")`. This way it does not have to be looked up in every iteration.

Also do `chommentRootViewResId` for consistency.

```suggestion
                if (nestedChild.getId() == chatMessageItemResId) {
```